integrations/google_vertex/src/haystack_integrations/components/generators/google_vertex/chat/gemini.py

```diff
@@ -1,14 +1,15 @@
 import logging
-from typing import Any, Dict, List, Optional, Union
+from typing import Any, Callable, Dict, List, Optional, Union
 
 import vertexai
 from haystack.core.component import component
 from haystack.core.serialization import default_from_dict, default_to_dict
+from haystack.dataclasses import StreamingChunk
 from haystack.dataclasses.byte_stream import ByteStream
 from haystack.dataclasses.chat_message import ChatMessage, ChatRole
+from haystack.utils import deserialize_callable, serialize_callable
 from vertexai.preview.generative_models import (
```

Comment:
Also wondering, if we can use more stable `vertexai.generative_models` instead of preview.

Reply:
Out of the following imports
```
from vertexai.preview.generative_models import (
    Content,
    GenerationConfig,
    GenerativeModel, 
    HarmBlockThreshold,
    HarmCategory,
    Part,
    Tool,
)
```
only `GenerativeModel` has a different implementation in `vertexai/generative_models/_generative_models.py` and in `vertexai/preview/generative_models.py` all others point to the same implementation. So we could change the imports for those. For GenerativeModel you would need to please double check the differences in the implementation and changing the import for GenerativeModel would most likely be a breaking change.

Reply:
Will create an issue for this and we can handle this separately.